pwndbg/storage.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import os
+import sys
+
+def get_cache_dir():
+    dir = os.environ.get('PWNDBG_CACHE_DIR')
+    if not dir:
+        if sys.platform == 'darwin':
+            dir = os.path.join(os.path.expanduser('~/Library/Caches'), 'pwndbg')
+        elif sys.platform == 'win32':
+            dir = os.path.join(os.getenv('LOCALAPPDATA'), 'pwndbg', 'Cache')
```

Comment:
We probably should add a mechanism to memoize or save/load something in cache in this PR.

Just a note: it would be nice to have an assertion to error on saving two things into same file, not to have weird issues in the future.
This might be tricky considering the fact we might have code that reloads modules (see: https://github.com/pwndbg/pwndbg/pull/267).

Hm.. if it is going to be a decorator it could be implemented so that we would take import path + function name as cache key (or rather file name) - this won't have duplicates.